### ISSUE TITLE
Volume conversion providing wrong result. 

### ISSUE THREAD

**Issue by sajjadIslam2619** [HanSolo/converter#3]
```
Converter volumeConverter_m3 = new Converter(VOLUME, CUBIC_METER);
        double meter = 3.0;
        double liter = volumeConverter_m3.convert(meter, LITER);
        System.out.println("liter :: " + liter);
        System.out.println("liter :: " + BigDecimal.valueOf(liter).toPlainString());
```
This conversion showing result
liter :: 3.0000000000000005E-6
liter :: 0.0000030000000000000005

But from Google unit conversion it should be : 
![image](https://user-images.githubusercontent.com/7568561/91136017-03d38000-e6d0-11ea-959d-c3804f5638c0.png)

**Comment by HanSolo** [comment]
Fixed with commit 9f1e2dfe558f9ab4e15b24dd911652d88ce9806d. There was the wrong base unit set for volume which lead to the problem

**Comment by sajjadIslam2619** [comment]
Thanks :)
